inc/common/purge.php

```diff
@@ -419,23 +419,24 @@ function rocket_post_purge_urls_for_qtranslate( $urls ) {
  */
 function do_admin_post_rocket_purge_cache() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals
 	if ( isset( $_GET['type'], $_GET['_wpnonce'] ) ) {
-		$type = sanitize_key( $_GET['type'] );
+		$type_raw   = sanitize_key( $_GET['type'] );
+		$type_array = explode( '-', $type_raw );
 
-		$_type     = explode( '-', $type );
-		$_type     = reset( $_type );
-		$_id       = explode( '-', $type );
-		$_id       = end( $_id );
-		$_taxonomy = isset( $_GET['taxonomy'] ) ? sanitize_title( wp_unslash( $_GET['taxonomy'] ) ) : false;
+		$type     = reset( $type_array );
```

Comment:
Why is `reset()` necessary here? With the `explode()` it returns an array where the pointer is set to the first element in that array.

Reply:
Removed in https://github.com/wp-media/wp-rocket/pull/2616/commits/cbd10f9a7146bc28cb754b392f427092070b9a7c